setup.py

```diff
@@ -352,7 +347,7 @@ def load(filename):
         _cmake_build_dir, "deps", "magnum-bindings", "src", "python"
     )
 
-    if not has_magnum:
+    if not args.skip_reinstall_magnum and not is_pip():
         subprocess.check_call(shlex.split(f"pip install {pymagnum_build_dir}"))
     else:
         print("Assuming magnum bindings are already installed")
```

Comment:
```suggestion
         print("Assuming magnum bindings are already installed (or we're inside pip and ¯\\_(ツ)_/¯)")
```

I would still yell *a bit*, tho -- better to have the message overly verbose than hiding something important. Feel free to change it to something reasonable, of course :)

Reply:
Dang *why* it made that extra indentation there :see_no_evil: GitHub, you're buggy.

Reply:
Seems like the unicode character isn't being parsed correctly :(

Reply:
oh well, but that's a normal katakana character? is this python 2 or...? :D

Reply:
It’s 3.6 so who knows